go-backend/db/recognition_test.go

```diff
@@ -0,0 +1,74 @@
+package db
```

Comment:
we have deprecated this strategy for writing test case, please refer https://github.com/joshsoftware/peerly/pull/110
for the new strategy

Reply:
Its already merged so the code is in master also